packages/expo-cli/src/commands/eject/Eject.js

```diff
@@ -206,16 +206,19 @@ from \`babel-preset-expo\` to \`babel-preset-react-native-stage-0/decorator-supp
     pkgJson.scripts.ios = 'react-native run-ios';
     pkgJson.scripts.android = 'react-native run-android';
 
-    if (pkgJson.jest && pkgJson.jest.preset === 'jest-expo') {
-      pkgJson.jest.preset = 'react-native';
-      newDevDependencies.push('jest-react-native');
-    } else if (pkgJson.jest) {
-      log(
-        `${chalk.bold(
-          'Warning'
-        )}: it looks like you've changed the Jest preset from jest-expo to ${pkgJson.jest
-          .preset}. We recommend you make sure this Jest preset is compatible with ejected apps.`
-      );
+    if (pkgJson.jest !== undefined) {
+      pkgJson.scripts.test = 'jest';
+
+      newDevDependencies.push('jest');
+
+      if (pkgJson.jest.preset === 'jest-expo') {
+        pkgJson.jest.preset = 'react-native';
+        newDevDependencies.push('jest-react-native');
```

Comment:
I think we can remove this line since the preset is `react-native` instead of `jest-react-native` now?